source/MicroBit.cpp

```diff
@@ -121,7 +121,7 @@ void MicroBit::init()
     // Create an event handler to trap any handlers being created for I2C services.
     // We do this to enable initialisation of those services only when they're used,
     // which saves processor time, memeory and battery life.
-    messageBus.listen(MICROBIT_ID_MESSAGE_BUS_LISTENER, MICROBIT_EVT_ANY, this, &MicroBit::onListenerRegisteredEvent);
+    messageBus.listen(MICROBIT_ID_ANY, MICROBIT_EVT_ANY, this, &MicroBit::onListenerRegisteredEvent);
```

Comment:
Joe suggests creating a new `MICROBIT_ID_SYSTEM` event to prevent churn
- Things like the accelerometer will constantly be firing events at this
- A system event could be used for other things

Reply:
MICROBIT_ID_MESSAGE_BUS_LISTENER renamed to SYSTEM